packages/shared/components/pills/Pill.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+    import { Text, FontWeight, TextType } from 'shared/components'
+
+    export let data: string | undefined = undefined
+    export let textColor: string = 'gray-800'
+    export let darkTextColor: string | undefined = undefined
+    export let shrink: boolean = false
+    export let uppercase: boolean = false
+    export let invisible: boolean = false
+    export let backgroundColor: string | undefined = undefined
+    export let rounded: string = 'rounded-2xl'
+</script>
+
+<pill-wrapper
+    class:shrink-0={shrink}
+    class:uppercase
+    class:invisible
+    class="flex items-center {backgroundColor ?? ''} {rounded} px-2.5 py-1"
```

Comment:
how about

```suggestion
    class:invisible
    class:rounded-full={rounded}
    class="flex items-center {backgroundColor ?? ''} rounded-2xl px-2.5 py-1"
```

or maybe call it `roundedFull`

Reply:
mm maybe I could rename it to `borderRadius`, wdyt? so we can use other classes like `rounded-md`, `rounded-xl`...

Reply:
Actually we don't even need any of these. If necessary I think we could do a wrapper like 
```
<div class="rounded-full overflow-hidden">
    <Pill
       ...
</div>

```

Reply:
as we discussed on Slack, I've added an enum :pray: